From 32ce32b0542381a49cbf7c147d8fb0d5ba7a2d1d Mon Sep 17 00:00:00 2001
From: Gregory Chanan <gchanan@cloudera.com>
Date: Tue, 18 Dec 2012 12:26:45 -0800
Subject: [PATCH 57/96] CDH-8548 Turn 92/94 compatibility mode on by default in CDH4.2

Reason: Compatibility
Author: Gregory Chanan
Ref: CDH-8548

(cherry picked from commit 1a8bed9bcd604519cdbeb31601eb7f59687eebf8)
---
 .../hadoop/hbase/zookeeper/ZooKeeperWatcher.java   |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/main/java/org/apache/hadoop/hbase/zookeeper/ZooKeeperWatcher.java b/src/main/java/org/apache/hadoop/hbase/zookeeper/ZooKeeperWatcher.java
index 0274db9..75d9c37 100644
--- a/src/main/java/org/apache/hadoop/hbase/zookeeper/ZooKeeperWatcher.java
+++ b/src/main/java/org/apache/hadoop/hbase/zookeeper/ZooKeeperWatcher.java
@@ -214,11 +214,11 @@ public class ZooKeeperWatcher implements Watcher, Abortable {
         conf.get("zookeeper.znode.unassigned", "unassigned"));
     String tableZNodeDefault = "table";
     masterTableZNode = ZKUtil.joinZNode(baseZNode,
-        conf.get("zookeeper.znode.masterTableEnableDisable", tableZNodeDefault));
+        conf.get("zookeeper.znode.masterTableEnableDisable", "table94"));
     clientTableZNode = ZKUtil.joinZNode(baseZNode,
             conf.get("zookeeper.znode.clientTableEnableDisable", tableZNodeDefault));
     masterTableZNode92 = ZKUtil.joinZNode(baseZNode,
-        conf.get("zookeeper.znode.masterTableEnableDisable92", "table92"));
+        conf.get("zookeeper.znode.masterTableEnableDisable92", tableZNodeDefault));
     clusterIdZNode = ZKUtil.joinZNode(baseZNode,
         conf.get("zookeeper.znode.clusterId", "hbaseid"));
     splitLogZNode = ZKUtil.joinZNode(baseZNode,
-- 
1.7.0.4

